packages/ejson/ejson.js

```diff
@@ -131,10 +131,10 @@ var builtinConverters = [
   },
   { // Escaping one level
     matchJSONValue: function (obj) {
-      return _.has(obj, '$escape') && _.size(obj) === 1;
+      return _.has(obj, '$escape') && hasSize(obj, 1);
     },
     matchObject: function (obj) {
-      if (_.isEmpty(obj) || _.size(obj) > 2) {
+      if (!hasSize(obj, 1)) {
```

Comment:
I don't think this is the same functionality, right? The code before worked for sizes 1 and 2 (unless that was a bug)